api/jobs/bot.py

```diff
@@ -45,9 +42,7 @@ def get_ides_opened_this_semester() -> int:
 
     :return: Total number of IDEs opened this semester
     """
-    return TheiaSession.query.join(
-        Course, TheiaSession.course_id == Course.id
-    ).count()
+    return TheiaSession.query.join(Course, TheiaSession.course_id == Course.id).count()
```

Comment:
I completely understand why you want to make this one line. The problem is that this line now has more than 80 characters, which is generally considered a bad coding style. If you check out other scripts in `api` you'll see similar statements or expressions which could be written in one line but are written in multiple lines just to follow this 80 characters rule. So please fix all the changes you made that breaks this rule. Thanks.

Reply:
I didn't do so; I ran black on the file for linting, which formatted it like that.

Reply:
We've been using 120 as the max line length. So there is no problem with this line